app/helpers/application_helper.rb

```diff
@@ -62,13 +62,13 @@ def title_with_timeframe(page_title:, timeframe:, content_for: false)
     content_for ? title(title_text) : title_text
   end
 
-  def optimized_image_url(url, width: 500, quality: 80, fetch_format: "auto", random_fallback: true)
+  def optimized_image_url(url, width: 500, quality: 80, fetch_format: "auto", random_fallback: true, radius: nil)
     fallback_image = asset_path("#{rand(1..40)}.png") if random_fallback
 
     return unless (image_url = url.presence || fallback_image)
 
     normalized_url = Addressable::URI.parse(image_url).normalize.to_s
-    Images::Optimizer.call(normalized_url, width: width, quality: quality, fetch_format: fetch_format)
+    Images::Optimizer.call(normalized_url, width: width, quality: quality, fetch_format: fetch_format, radius: radius)
```

Comment:
I added this `radius` param that is handled correctly by the [cloudinary helper tag](https://cloudinary.com/cookbook/applying_rounded_corners_to_an_image) in order to round the corners of the logo. I'm not too sure how to test this for the `imgproxy` side of things though or if there would be problems with this solution. There's a chance we'll need to add [a translation for this param](https://github.com/forem/forem/blob/master/app/services/images/optimizer.rb#L48-L54).

@maestromac since you've worked with `imgproxy` the most maybe you could point me in the right direction here?

If this is not the best solution for the problem it's totally okay if we need to close this PR in favor of a different approach

Reply:
Imgproxy is only for image resizing and image conversion. It cannot do any advanced edits, like giving border-radius. So if you were to add this, it would only work for DEV. 

My rough take on this is it's might be better to provide another Siteconfig image field, so the users won't be surprised by the hidden border-radius feature. 

You can play around with Imgproxy locally by following the doc [here](https://docs.forem.com/installation/imgproxy/)

Reply:
Makes sense thanks @maestromac! Closing the PR now as this is not a good approach based on that feedback